core-feature-pack/common/src/main/resources/content/bin/domain.ps1

```diff
@@ -113,4 +113,4 @@ if ($SERVER_OPTS -ne $null){
 $backgroundProcess = Get-Env LAUNCH_JBOSS_IN_BACKGROUND 'false'
 $runInBackGround = $global:RUN_IN_BACKGROUND -or ($backgroundProcess -eq 'true')
 
-Start-WildFly-Process -programArguments $PROG_ARGS -runInBackground $runInBackGround
+Start-WildFly-Process -programArguments $PROG_ARGS -runInBackground $runInBackGround -logFileProperties "$JBOSS_CONFIG_DIR\logging.properties" -logFile "$JBOSS_LOG_DIR\host-controller.log"
```

Comment:
Does this need to be added to the standalone.ps1 as well?

Reply:
@spyrkob Not really, both domain.ps1 and standalone.ps1 use a common method to kick off the server. This common method gets the argument with default values when standalone.ps1 invokes it. So, we only overwrite the defaults when using domain.ps1

See: https://github.com/wildfly/wildfly-core/blob/3aa4a9ee0720e1a4c07dfd1b5b9fd1def0355a00/core-feature-pack/common/src/main/resources/content/bin/common.ps1#L351-L353

Reply:
ah I missed that, thanks!